content/klc/S2.2.adoc

```diff
@@ -13,7 +13,3 @@ Examples:
 To capture every possible combination of part variation requires a large number of symbol aliases - this is to be avoided.
```

Comment:
I do not want to go into too much detail with the KLC as it is already too long. the fact that different footprints require different symbols is made abandonedly clear in other rules and does not need to be repeated here. The rest will highly depend on device type and is something that will mostly be dealt with in the pull requests. (I could however mention that temperature ranges, standard compliance and packaging variants should never be included.)

Reply:
I wouldn't consider the KLC too long (it's a long read, but when you have a specific question about drawing a part, details and examples are always helpful). In this case, there is a distinction made and having examples of both side of the distinction could help to make the distinction more clear (as well as provide more clear rules for any parts to which the examples apply directly). But I won't insist more than this :-)

Reply:
When a single manufacturer makes a part in one or more packages, the MPN is used. No generic suffix of the package type is required because there will be no conflicts.

Just above this line you selected in S2.2 are examples of things to wildcard. Footprints are naturally captured by the symbol and that's already covered. Voltage ratings are usually split out into ALIASes these days instead of wildcarding as per Rene's preference and to make part selection easier as those are functional differences. I don't know about accuracy but I would be inclined to wildcard it since that only limits the range at possible values and is usually a binned/tested characteristic, and not a design/functional difference.